spec/views/cases/cover_pages/show_html_slim_spec.rb

```diff
@@ -0,0 +1,37 @@
+require 'rails_helper'
+
+describe 'cases/cover_pages/show', type: :view do
+  context '#show' do
+    let(:data_request) {
+      create(
+        :data_request,
+        location: 'HMP Leicester',
+        request_type: 'all_prison_records',
+        date_from: Date.new(2018, 8, 15),
+        date_to: Date.new(2019, 8, 15),
+        cached_num_pages: 32,
+        cached_date_received: Date.new(1972, 9, 25),
+      )
+    }
+
+    before do
+      assign(:data_request, data_request)
+      assign(:case, data_request.kase)
```

Comment:
I think here you may need to create a case with a known ID for the test to pass.

Reply:
This is a pattern that we've use in existing specs - the factory for the data request creates the parent case I think - passed locally 🤔